### ISSUE TITLE
Ghosts can interact with/activate large xenoarch artifacts

### ISSUE THREAD

**Issue by AceDick** [d3athrow/vgstation13#2575]
This was an issue I thought was already listed, ghosts are able to interact with xenoarch artifacts, touching them as if they were humans with ungloved hands, leading to triggering of them that can range from inconvenient and spammy to devestating if it's a bad effect.

**Comment by ComicIronic** [comment]
Should be fixed after #2579.

Also check harder before making issues, this was a duplicate of #1645.
